docs/CheckForApplicationUpdate.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: default
+title: "The CheckForApplicationUpdate component"
+---
+
+# `CheckForApplicationUpdate`
+
+This component regularly checks whether the application source code has changed and prompt users to reload the page when an update is available. To detect updates, it fetches the current URL at regular intervals and compares a hash of the response content.
+
+It leverages the [`useCheckForApplicationUpdate`](./useCheckForApplicationUpdate.md) hook.
+
+![CheckForApplicationUpdate](./img/CheckForApplicationUpdate.png)
+
+## Usage
+
+Include this component in a custom layout:
+
+```tsx
+// in src/MyLayout.tsx
+import { CheckForApplicationUpdate, Layout, LayoutProps } from 'react-admin';
+
+export const MyLayout = ({ children, ...props }: LayoutProps) => (
+    <Layout {...props}>
+        <CheckForApplicationUpdate />
+    </Layout>
+);
+
+// in src/App.tsx
+import { Admin, ListGuesser, Resource } from 'react-admin';
+import { MyLayout } from './MyLayout';
+
+export const App = () => (
+    <Admin layout={MyLayout}>
+        <Resource name="posts" list={ListGuesser} />
+    </Admin>
+);
+```
+
+## Props
+
+`<CheckForApplicationUpdate>` accepts the following props:
+
+| Prop            | Required | Type     | Default            | Description                                                         |
+| --------------- | -------- | -------- | ------------------ |-------------------------------------------------------------------- |
+| `checkInterval` | Optional | number   | `3600000` (1 hour) | The interval in milliseconds between two checks                     |
+| `disabled`      | Optional | boolean  | `true`             | Whether the automatic check is enabled                              |
```

Comment:
very confused about whether this is `disabled` or `enabled`, either the prop name or the default value and the description are wrong

Reply:
Besides, I don't really understand the need to disable the feature by default, since, by default, the Layout won't include this component

Reply:
The doc is wrong